platform/docs/docs/configuration/configuration.md

```diff
@@ -59,6 +73,38 @@ window.config = {
 };
 ```
 
+## Run Time Configuration (Config-Point)
+There is a library [config-point](https://github.com/OHIF/config-point)
+used to allow loading of configuration values dynamically,
+that is, at load time rather than being built into the runtime configuration.
+A user of OHIF can specify a dynamic configuration by adding one or more theme
+parameters, for example:
+```
+https://ohif.hospital.org/?theme=mgHP&theme=euroKeyboard
+```
+to load two hypothetical theme settings files mgHP and euroKeyboard to add
+mammographic hanging protocols and European keyboard settings.
+
+A site can add such settings by creating custom files in:
+```
+.../build/theme/mgHP.json5
```

Comment:
can we be more specific about where these themes should be located? where is `build` located? how to experiment with it in the `development`